versioned_docs/version-0.x.0/kubeslice-cli/kubeslice-cli-commands/kubeslice-cli-commands.mdx

```diff
@@ -0,0 +1,54 @@
+# KubeSlice CLI Commands
+
+This section contains information about KubeSlice commands, syntax, options, a resource type, and a few examples of command usage.
+
+## Syntax
+Use the following syntax to run the kubeslice-cli tool:
+
+```
+kubeslice-cli [global options] <command> <command-options>
+```
+
+## Commands
+
+The following table describes the syntax and descriptions for all the `kubeslice-cli` commands.
+
+|Operations|Syntax|Description|
+|----|----|---|
+| create | `kubeslice-cli create [flags]` | Creates one or more resources from a file. |
```

Comment:
replace flags with `options` in this entire table...

Reply:
done

Reply:
Have you changed them in your local and not pushed them here?